cmd/fleetctl/package.go

```diff
@@ -220,11 +221,29 @@ func packageCommand() *cli.Command {
 				zlog.Logger = zerolog.Nop()
 			}
 
-			fmt.Println("Generating your osquery installer...")
-			path, err := buildFunc(opt)
+			const maxAttempts = 9 // see #5732
```

Comment:
I was thinking about this topic recently, we also:

1. want to add [retries for notarization](https://github.com/fleetdm/fleet/issues/7130) (as we have to upload `pkg`s to Apple servers and sometimes the network fails)
2. have retry logic for launching Fleet desktop
3. have retry logic for some network tests

At which point should we think of adding a generic retry package? I got started on something on the lines of:

```go
type config struct {
       retryInterval time.Duration
       waitFirst     bool
       done          chan struct{}
       maxAttempts   int
}

func Do(fn func() error, opts ...Option) error { /* ... */ }
```

but I'm not sure if it's even worth it, as we have to touch Orbit